src/pages/Confirm.vue

```diff
@@ -67,27 +105,65 @@ onMounted(async () => {
       redirectToResult(jsonrpc, { message: "Invalid or Missing Params", method }, req_id, resolveRoute);
       return;
     }
+
     origin.value = txData.origin as string;
     network.value = txData.network || "";
+    const networkConfig = txData.networkDetails as ProviderConfig;
```

Comment:
`txData.networkDetails` is not defined for redirect flow, use wallets providerConfig if txData doesn't have it.